HW0/P6/P6B.py

```diff
@@ -16,16 +17,27 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = [10**(-i) for i in xrange(7)]
 
     for t in wait_time:
         # Compute jobs serially and in parallel
         # Use time.time() to compute the elapsed time for each
-        serialTime = 1
-        parallelTime = 1
+
+        # Serial
+        start = time.time()
+        pool.map(burnTime(t), range(N))
```

Comment:
I may be wrong, but shouldn't the second parameter to map() be the values burnTime is operating on? So what I did was pool.map(burnTime,[t,t,t,t,t,t,t,t,t,t,t,t,t,t,t]) where t is the wait time, and I would do this for each wait time.